packages/build/src/plugins_core/deploy/buildbot_client.js

```diff
@@ -0,0 +1,86 @@
+const net = require('net')
+const { promisify } = require('util')
+
+const pEvent = require('p-event')
+
+const { addErrorInfo } = require('../../error/info')
+
+const startBuildbotClient = async function(buildbotServerSocket) {
+  if (buildbotServerSocket === undefined) {
```

Comment:
`buildbotServerSocket` should never be `undefined` here since the plugin would not be enabled without the CLI flag.